src/rhub/api/utils.py

```diff
@@ -13,6 +17,12 @@ def to_dict(self):
             data[column.name] = getattr(self, column.name)
         return data
 
+    def to_dict_with_super(self) -> dict[str, str]:
```

Comment:
You can change the `to_dict` method, I think it won't break anything. I already noticed that columns from the parent class are not serialized to dict, it is bug. I wanted to fix it, but it disappeared in my git stash so I forgot...

Reply:
Thanks for clearing that. I'll put that code on `to_dict` in other PR, to ease the backout case we need it.

Reply:
https://issues.redhat.com/browse/EXDRHUB-959